client/overview.jsx

```diff
@@ -34,7 +34,13 @@ class Overview extends React.Component {
 
 
   fetchOverview() {
-    return axios.get('http://localhost:3003/overview/all', { params: { campId: this.state.campId } })
+    const url = {}
+    if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
```

Comment:
ooh what does this do??

Reply:
it changes the url per the node deployment so that I dont have to do it everytime... lol